c2corg_api/models/route.py

```diff
@@ -322,3 +322,7 @@ class ArchiveRouteLocale(_RouteLocaleMixin, ArchiveDocumentLocale):
     'locales.title', 'locales.title_prefix', 'elevation_min', 'elevation_max',
     'activities'
 ])
+schema_association_waypoint_route = restrict_schema(schema_route, [
+    'locales.title', 'locales.title_prefix', 'elevation_min', 'elevation_max',
+    'activities', 'geometry.geom_detail'
```

Comment:
Concerning the route ``geom_detail`` we should probably do something to avoid huge GPS tracks, that could be a problem event in the route detail view page. For instance:
http://c2corgv6.demo-camptocamp.com/routes/211687/fr
http://www.camptocamp.org/routes/211687/fr/mont-de-grange-traversee-bel-air-coicon
=> the GPX track in the v5 page is > 800 kB